installer.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+SOURCE_DIR="/var/tmp/sources-install/"
+REQUIREMENTS_FILE="/var/tmp/actual-reqs.txt"
+
+mkdir -p $SOURCE_DIR
+
+pip install --upgrade pip
+
+pip download -r $REQUIREMENTS_FILE -d $SOURCE_DIR
```

Comment:
I guess this could actually work really well if we had a downstream pip mirror.....

Reply:
working on that, or something similar ;)
but there's a way to get the tarballs downstream without using pip

Reply:
can we also mirror requirements and use upper-constraints?

Reply:
that's the plan :)